docs/docsite/rst/community/steering/steering_committee_membership.rst

```diff
@@ -84,23 +84,26 @@ A Committee member will be removed from the Committee if they:
 Absence or irregular participation in discussing topics and votes
 ..................................................................
 
-In case of absence or irregular participation, the involuntarily removal process consists of the following steps:
+In case of absence or irregular participation, the removal process consists of the following steps:
 
 #. Another Committee member (hereinafter the initiator) contacts the person by email asking if they are still interested in fulfilling their Committee member's duties.
-#. If they respond that they are not interested, the initiator asks the person to step down on their own following the :ref:`Voluntarily leaving process<Voluntarily leaving process>`.
-#. If there has been no response or stepping down issue created by the person within a reasonable time, the initiator notifies other Committee members about the situation.
-#. In case of agreement among the Committee about the need for removal, the initiator provides a draft of a corresponding topic's description to the Committee by email for discussion and approval.
 
-  * The topic's title is ``Steering Committee member audit.``. It must not contain the person's name or other identifying information.
+  * If the answer is negative, the initiator asks the person to :ref:`step down volunterily<Voluntarily leaving process>`.
 
-  * The description must not contain or imply any form of condemnation.
+#. In case there is no response from the person within a week after the email was sent or if the person agreed to step down but has no time to do it themselves, the initiator:
 
-  * It must mention that the person has been inactive for an unknown reason for the last N months and that, in accordance with the Steering Committee policies, their place should be freed for another person who can continue their great job.
+  * Sends a ``message`` to the ``SteeringCommittee`` group on the forum.
 
-  * The description must mention person's achievements and thanks for their time and effort they spent serving for the Community, Committee, and the Project, and a hope that one day they will come back.
+  * The message title is ``Steering Committee member audit.``. It must not contain the person's name or other identifying information.
```

Comment:
> It must not contain the person's name or other identifying information.

I know this was here before, but I'm still not sure what it means. How can we create an issue about someone without identifying them?

Reply:
By Forum/GH login i think but anyway now it feels not that relevant.
So let's remove this

Reply:
done, PTAL